app/services/plan_service/store/plan.rb

```diff
@@ -2,20 +2,26 @@ module PlanService::Store::Plan
 
   class TrialModel < ActiveRecord::Base
     self.table_name = :marketplace_trials
+    serialize :features, Array
```

Comment:
@rap1ds one of the ideas behind replacing `plan_level` with `features` is to simplify filtering the correct features for a given plan. With `features` field, the same logic can be applied to all plans regardless of the plan type.